### ISSUE TITLE
Uninstall source version

### ISSUE THREAD

**Issue by tavinus** [simoncadman/CUPS-Cloud-Print#109]
I can't find any detailed info on how to uninstall if using `./configure`, `make`, `make install`.  
I wanted to remove the source version to use the package one. 

Having an uninstall target in the makefile would be nice. 

How should I proceed? 

Removing the `/usr/local/share/cloudprint-cups` folder is enough?

**Comment by virtualdll** [comment]
Yes. Linux doesn't have registry and apps are contained within their directory when done correctly. The only thing else outside of this would be logs but you can leave them. 

Thanks,
David Lamb

**Comment by tavinus** [comment]
Thanks for the reply.  
I ended up updating the source version, but my problem was really on the HP end (their server was down or at least not communicating as it should).  
If the uninstall procedure is really to just remove the `/usr/local/share/cloudprint-cups`, then I may as well add it as a target to the makefile and make a pull request. 

Makes me wonder if I should delete all printers from CUPS before doing this though.  

Also wondering if I should remove the config file in `/etc` as well. 

I am still not sure if just removing a printer from `localhost:631` (cups) is enough for uninstalling cloud printers as well. For example, the queue for that printer may remain installed and active.

I guess I will make some tests on a virtual machine. Cheers!
